clients/fossid-webapp/src/main/kotlin/FossIdRestService.kt

```diff
@@ -43,18 +44,19 @@ import retrofit2.http.POST
 
 interface FossIdRestService {
     companion object {
+        val FOSSID_JSON_MAPPER: ObjectMapper = JsonMapper()
```

Comment:
Please omit the superfluous type.

Reply:
I put it because Intellij has a warning :
```
Declaration has type inferred from a platform call, which can lead to unchecked nullability issues. Specify type explicitly as nullable or non-nullable.

This inspection reports functions and properties that have a platform type. In order to prevent unexpected errors, the type should be declared explicitly.
```

Reply:
Thanks for the note. You can apply a small trick here and call `.registerKotlinModule()` last. As that is a Kotlin extension function which already applies the correct type, you can then omit it here 😁

Reply:
Good idea !